app.js

```diff
@@ -410,24 +442,42 @@ const tslint = {
     printDone('Installing HammerJS...');
   }
 
+  if (answers.git) {
+    printMsg('Linkining to git...');
+    try {
+      version = cp.execSync(`git --version`);
+    } catch (error) {
+      printDone('git is not installed...');
+    }
+    if (version && !!version.toString()) {
+      cp.execSync(`git init`);
+      fs.writeFile('./.gitignore', 'node_modules', () => {});
```

Comment:
same comment, angular creates a .gitignore file by default which is already packed with nodejs and angular stuff to ignore